SR2025/2024-09-02-competition-announcement.md

```diff
@@ -0,0 +1,36 @@
+---
+to: Potential team supervisors
+subject: Announcing Student Robotics 2025!
+---
+
+Hi,
+
+We're pleased to [announce][announcement] that registration for Student Robotics
+2025 is now open. If you would like to compete, please complete the
+[signup form][signup-form] with the details of your team. All future emails
+about SR2025 will be sent to the details you provide there.
+
+This year we're hosting [Kickstart][kickstart] at the University of Southampton,
+on Saturday 12th October. We will also be live-streaming the event for those
+unable to attend in person. We expect to confirm places towards the latter half
+of September. Places are limited, so sign up soon to avoid disappointment.
+
+As usual we will loan competing teams a [kit][kit] of parts to help build their
+robots. This year we hope to send out kits before Kickstart.
+
+We have also prepared some [pre-Kickstart activities][pre-kickstart-activities]
```

Comment:
We could highlight the pre kickstart activities with some emphasis on the fact they are new so they are even less likely to be missed?
```suggestion
Also **new for this year** we have prepared some [pre-Kickstart activities][pre-kickstart-activities]
```